api/local.js

```diff
@@ -0,0 +1,72 @@
+const http = require('http');
```

Comment:
For development purposes only! I'll remove this file once functional tests are in place.